### PR TITLE
Development edit of self-care chapter.

### PR DESCRIPTION
**Pull request by bproffitt**
Development edit of chapter, #70

### CODE REVIEW

**Review by quaid — APPROVED** [review]
These are not substantive changes to the content but cover mainly formatting and style, and perhaps some American English choices. I'm going to accept these changes on behalf of @samus-aran, who is on leave right now, as I think these move the content along without needing the original author approval.

**Comment by quaid** [comment]
I'm merging this PR as the changes are not substantive but more formatting and style. Since the author is on leave for the time being, I'm going to be a bold editor and move this forward.

File: community-manager-self-care.adoc
Comment (samus-aran) [inline community-manager-self-care.adoc:506]: Agreed and added to the to-do list.

**Comment by samus-aran** [comment]
Thanks for this everyone. Although it has already been merged I completely agree with this review and will create a separate PR to the action points raised :+1:

**Comment by quaid** [comment]
> Thanks for this everyone. Although it has already been merged I completely agree with this review and will create a separate PR to the action points raised +1

Thank you! And thanks for understanding my merging your PR before your review, I was getting confused in hat-wearing (author? editor? take charge and get it done-er?) and also wanted to move things along.

I will need to do a copyedit and/or style review to make sure that things align with the rest of the general formatting we're doing; which I'll set up as a non-blocking pull request.

**Comment by samus-aran** [comment]
> Thank you! And thanks for understanding my merging your PR before your review, I was getting confused in hat-wearing (author? editor? take charge and get it done-er?) and also wanted to move things along.

I think that this is a good example of an acceptable situation when an author goes AWOL (even if temporarily) and how we can retrospectively have alterations to reviews when or if the author returns. An editor of the core team should have the authority to merge format and non substantial changes IMO for the benefit of the project. Utilising the "agile" methodology in the editorial process :smile:

**Pull request merged**
